### DISCUSSION TITLE
Not working for me

### DISCUSSION THREAD

**Discussion by Harley-Jones** [ImageMagick/ImageMagick#6876] [Help]
### ImageMagick version

ImageMagick-7.1.1-21-Q16-x64-dll.exe

### Operating system, version and so on

Windows 10 home pro 22H2 19045 64gb ram lga1151 Core i7 7700k Kabby Lake Gtx 1070 

### Description

So I have been trying now for several months to get imagemagic to work. I'm not sure what everyone else uses it for primarily as it sounds like quite the sophisticated piece of software, but I require it as a resource That Rain Meter uses in order to facilitate certain functions of a skin theme environment called JaxCore. It's essentially a very well designed set of widgets and tools that allow you to customize your desktop experience. But no matter what I seem to do I end up with a new problem. even if that problem is that the install goes fine and the program works on the computer, then as luck would have it it still isn't recognized as installed by rain meter. do to this I am unable to edit skins and various widgets will not work. any help would be greatly appreciated. I am happy to use quick assist program If that makes its easier and faster to administer assistance. thanks in advance hope everyone else is doing great.

**Comment by fmw42** [comment]
Just a guess, but does your Rainmaker require Imagemagick 7 or Imagemagick 6.  You are perhaps trying to install Imagemagick 7 and it wants Imagemagick 6.  The windows installer will install Imagemagick 6 if you select the legacy components with your Install.  Check to see which version Rainmaker actually wants.

**Comment by Harley-Jones** [comment]
Their website link with instructions that I followed to the letter direct
you to imagemagic7

**Reply by dlemstra** [reply]
I would advise you to reach out to the That Rain Meter team instead then.

**Comment by Harley-Jones** [comment]
Ok ill reach out to them and see what they say, even though we nobody
suggested anything I could try on my end to verify that it isnt a problem
image magic is having. It worked just fine untill several months ago when i
reisntalled my Windows  OS. You wouldn't believe me if i told you why. The
error I get is:

IMDisplayDoc function [DoReadImage] reported an error.
Imdisplay.exe: no decode delegate for this image format 'INI' @
error/constitute.c/reanimated/746

I've attached a photo of what it does, it's actually really neat and barely
effects the number of background processes and system resource use.

**Reply by dlemstra** [reply]
This looks like a bug in their software, where they try to read an `.ini` file as an image.

**Comment by Harley-Jones** [comment]
When I saw ini. Myvfirst thought was why do you need and image program to
open a text file?I'm in like week three of learning Javascript so I'm not
quite to the point where the picture has all come together like it has for
ll of you
